Health_And_Safety.c

```diff
@@ -5,7 +5,10 @@
 #include "Health_And_Safety.h"
 
 int check_Battery_Life(){
-
+// HI Jacob, this is David. I didn't see any comment so I just randomly chose this one. Hope it'll work!
+    int batteryPercent = getBatteryPercent(); 
```

Comment:
great job!  unfortunately, you and Ryan and Caleb all picked the same function to work on (my fault) so im just going to close this request, but I'm glad that you get everything figured out!